lib/cache/ObservableCollection.js

```diff
@@ -153,7 +157,7 @@ export default class ObservableCollection {
         if (this.matcher) {
             return !!this.collection.findOne(
                 _.extend({}, this.selector, { _id }),
-                { fields: { _id: 1 } }
+                { projection: { _id: 1 } }
```

Comment:
How is the backward compatibility on this one?

Reply:
Ok, let's use `fields` then.

Reply:
Well, that would be not fix the problem that this PR is trying to fix. I'm just trying to figure out how to have both available here, depending on which one is supported.

Reply:
Meteor will support 'fields' for some time, so we can still use it.

Reply:
Alright! I will release a patch version with these.